### PR TITLE
Feed bugs

### PR DESCRIPTION
**Pull request by sebdraven**

### CODE REVIEW

File: plugins/feeds/public/hybrid_analysis.py [hunk 1]
```diff
@@ -116,29 +118,38 @@ def analyze(self, item):
                 sha256_new_file = sha256.SHA256(value=extracted_file["sha256"]).save()
 
                 new_file.link_to(sha256_new_file, "sha256", self.name)
+                
+                path_extracted_file = None
+                if extracted_file["file_path"] and isinstance(extracted_file["file_path"], str):
```
Comment (tomchop) [inline plugins/feeds/public/hybrid_analysis.py:123]: Didn't you mean to check the presence of `file_path` in `extracted_file` here like in other changes?

**Review by tomchop — COMMENTED** [review]
A few nits / questions

File: plugins/feeds/public/hybrid_analysis.py [hunk 2]
```diff
@@ -116,29 +118,38 @@ def analyze(self, item):
...
                     new_file.tag(extracted_file["type_tags"])
+                    if path_extracted_file:
+                        path_extracted_file.tag(extracted_file["type_tags"])
+                    
+                    sha256_new_file.tag(extracted_file["type_tags"])
 
                 new_file.add_context(self.name, context_file_dropped)
                 sha256_new_file.add_context(self.name, context_file_dropped)
-
+
```
[the remark this answers is not in the document]
Comment (sebdraven) [reply plugins/feeds/public/hybrid_analysis.py]: Ok !

**Review by udgover — APPROVED** [review]
LGTM! Nice work to handle all hybrid analysis discrepancies.

**Pull request merged**